docs/tools/libzbd.md

```diff
@@ -9,52 +9,57 @@ import Video from '/src/components/Video';
+*libzbd* is a user library that provides functions for manipulating zoned block
 devices.
 
 Unlike the [*libzbc*](./libzbc) library, *libzbd* does not implement direct
-command access to zoned block devices. Rather, *libzbd* uses the kernel provided
-zoned block device interface based on the *ioctl()* system call. A direct
-consequence of this is that *libzbd* will only allow access to zoned block
-devices supported by the kernel running. This includes both physical devices
-such as hard-disks supporting the ZBC and ZAC standards, as well as all logical
-block devices implemented by various device drivers such as
-[*null_blk*](../getting-started/nullblk) and [device mapper](../linux/dm)
-drivers.
+command access to zoned block devices. *libzbd* instead uses the
+kernel-provided zoned block device interface that is based on the *ioctl()*
+system call. A direct consequence of this is that *libzbd* allows access only
+to zoned block devices that are supported by the running kernel. This includes
+both physical devices (such as hard-disks that support the ZBC and ZAC
+standards) and all logical block devices that are implemented by various device
+drivers (such as [*null_blk*](../getting-started/nullblk) and [device
+mapper](../linux/dm) drivers).
 
 The *libzbd* project is hosted on <a href="https://github.com/westerndigitalcorporation/libzbd"
 target="_blank">GitHub</a>.
 The project <a href="https://github.com/westerndigitalcorporation/libzbd/blob/master/README.md"
 target="_blank">*README* file</a> provides information on how to compile and 
-install *libzbd* library and its tools.
+install the *libzbd* library and its tools.
 
 ## Overview
 
-*libzbd* provides functions for discovering and managing the state of zones of
-zoned block devices. Read and write accesses to the devices can be done using
-standard standard I/O system calls.
+*libzbd* provides functions for discovering and managing the state of the zones
+in zoned block devices. Read and write accesses to the devices can be made
+using standard standard I/O system calls.
 
-The execution of *libzbd* functions as well as of write operations to zones of a
-device may result in changes to the condition or attributes of the device zones
-(such as write pointer location in sequential zones). These changes are not
-internally tracked by *libzbd*. In other words, *libzbd* is stateless. It is the
-responsibility of applications to implement tracking of the changes to zones
-conditions such as the increasing write pointer position of a sequential zone
-after the completion of a write request to the zone.
+When *libzbd* functions (and write operations) are performed on the zones of a
+zoned block device, the execution of those functions and operations can result
+in changes to the condition of the device zones and changes to the attributes
+of the device zones. An example of this is the location of the write pointer in
+sequential zones.
+
+In other words, *libzbd* is stateless.
```

Comment:
You removed "These changes are not internally tracked by *libzbd*." which puts the "stateless" statement out of context, so makes things hard to understand. Please do not drop that sentence.

Reply:
Wakatta.